### ISSUE TITLE
Systemd-boot log text in the background

### ISSUE THREAD

**Issue by davidv171** [cylgom/ly#79]
I'm on Arch and installed using the AUR 2 days ago. I've set:

# background and foreground truecolors
bg=0x2E3440
fg=0xD8DEE9

into the config to avoid getting this problem. I can't screenshot, but it basically looks like systemd-boot output logs(text) writing over the center box and the background. Save for setting up quiet boot is there any way of remedying this?

**Comment by cylgom** [comment]
Can you try the following?
```
force_update=1
blank_box=1
```
Optionally, you can also make the timeout shorter
```
min_refresh_delta=10
```

**Comment by davidv171** [comment]
Nope, nothing different using those. The text still overwrites the inner box and the background.

**Comment by JMasak** [comment]
I have the same problem. E.g. the message that the TLP service was started and some others were written over the login screen. 
As an intermediate workaraound i added tlp.service to AFTER=... in /usr/lib/systemd/system/ly.service.

**Comment by cylgom** [comment]
I just added a new "anmation" that blanks the whole screen.
To set it up, use something like this in your config.ini
```
# event timeout in millis
min_refresh_delta=10
# blank the background of the main box on update
blank_box=1
# force update the screen on event timeout
force_update=1
# use a background animation
animate=1
```

**Comment by jem014** [comment]
I wanted to inform you, that this problem still exists. I'm on Manjaro, so basically on Arch too.
Although, @JMasak 's workaround fixed the problem for me, I'm not really sure if this is a good workaround. So could you please elaborate if there are any risks coming from this workaround? And if there aren't, could you please add it either to the package or to the README? Thank you.

BTW, I really really like Ly, so keep up the good work!

**Comment by cylgom** [comment]
I just pushed some changes to the blanking animation, can someone else try it and confirm it works ?
All you need is this is your config (should be on by default if you installed the new version with the makefile)
```
# event timeout in millis
min_refresh_delta=1000
# blank the background of the main box on update
#blank_box=1
# force update the screen on event timeout
force_update=1
# use a background animation
animate=1
```
@JMasak 's solution is very specific because it prevents interfering services to launch in parallel. It's not bad, but it may not always work. The fix in my last commit is evil, but should work everywhere.

**Comment by JMasak** [comment]
Sorry for the late reply. I reverted my workaround, pulled the latest version and activated the new animation. Unfortunately i got the systemd messages over the login screen again.

**Comment by LinArcX** [comment]
same issue. without any configuration. just run ly after install. logout, see the ly:) but when i switch between input to set username or password, some logs appear in the screen that says:
>[** ]A start job is running for wait for Network to be configured (1s / no limit)

**Comment by cylgom** [comment]
This is driving me nuts, it seems like the only way to avoid that is to use getty or change the kernel settings... I tried mulitple things to fix that but failed each time. Really sorry about that :/

**Comment by jem014** [comment]
I just got time to try the changes to the blanking animation in the post that you removed again.
And a combinations of these changes and @JMasak 's workaround actually seems to do the trick for me. I tried it a couple of times and no log text so far. I'll keep you updated if anything changes.

**Comment by GoldsteinE** [comment]
Same issue. `Type=idle` worked for me. @cylgom, don't you want to fix it that way?

**Comment by Translator5** [comment]
you guys talk about this issue?

![photo_2019-03-21_17-51-09](https://user-images.githubusercontent.com/9306861/54769666-f1633600-4c01-11e9-800d-9034296e7805.jpg)

**Comment by oxodao** [comment]
Indeed

**Comment by cylgom** [comment]
Fixed in 4ab41f0, thanks everyone :)

**Comment by Translator5** [comment]
Approved as working. I received the update yesterday in Manџаro Linux.
But since I cannot change the language of my whole DE. 
It is straight overridden by english - and as it was the only update I got - I would like to let you know. Maybe it's not magic.
